applications/SwimmingDEMApplication/python_scripts/DEM_coupled_pfem_fluid_dynamics_analysis.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.PfemFluidDynamicsApplication
+
+from pfem_fluid_dynamics_analysis import PfemFluidDynamicsAnalysis
+
+class DEMCoupledPFEMFluidDynamicsAnalysis(PfemFluidDynamicsAnalysis):
+
+    def __init__(self, model, parameters=None, pp=None):
```

Comment:
Here you should be passing the whole project_parameters and only pass to the parent the project_parameters["fluid_parameters"] bit. See [https://github.com/KratosMultiphysics/Kratos/blob/SDEM/debugging-and-cleaning-analysis/applications/SwimmingDEMApplication/python_scripts/DEM_coupled_fluid_dynamics_analysis.py](url).